MMTabBarView/MMTabBarView/MMTabBarView.m

```diff
@@ -226,7 +226,7 @@ - (NSSize)intrinsicContentSize
     if ([_style respondsToSelector:@selector(intrinsicContentSizeOfTabBarView:)])
 	{
 		if(_isHidden)
-				return NSMakeSize(NSViewNoIntrinsicMetric, 0);
+				return NSMakeSize(NSViewNoInstrinsicMetric, 0);
```

Comment:
Deprecated does not mean removed… I don’t feel Apple will remove this constant soon.

I guess a precompiler test could handle this (at least some kind of #pragma to ignore the warning)… I don’t think it’s worth the effort though, because we will probably drop OS prior to 10.11 before the constant definition is effectively removed.